applications/eostarter.md

```diff
@@ -0,0 +1,233 @@
+# EOS Network Foundation Grant Proposal
+
+- **Project Name: EOSTARTER**
+- **Team Name: Eostarter Ltd.** 
+- **EOS Payment Address: eostartereos** 
+- **Level: 3**
+
+## Project Overview
+
+EOSTARTER is a community-powered Incubator for new projects building on EOS. We are building a Gamified Launchpad Platform where investors can participate in early-stage projects and be part of their success. As part of our business model, we launch Token Presales and NFT Drops for teams and companies that build their products on EOSIO.
+
+### Overview
+
+- **Name:** EOSTARTER TokenSale Platform 
+
+- **Brief Description:** We are proposing to develop a Token Sale Smart Contract Platform that allows for the crowdsourcing of projects VIA a Token Sale that can be configured with different parameters like Staking Requirements, whitelisting requirements, token vesting, milestone-based funding, and more.
+
+- **Relationship to EOSIO:** At EOSTARTER, we believe that EOSIO is only as valuable as the products and applications built on top of it, and this is our main focus to incentivize the growth of the ecosystem. Is short: “Succesful EOSIO Projects means Succesful EOSIO”
+
+- **Reason for Interest:** Our team's founder has been involved in EOS since Genesis and has been dedicating most of his energy and focus since to growing the ecosystem and bringing in new projects to build on EOS. We believe a new era for builders and entrepreneurs is coming to EOSIO. If we can build a platform that facilitates new projects to launch and build a community while also promoting the use of EOS and other EOSIO Tokens, then we all win!
+
+### Project Details
+
+The core details for the TokenSale Platform have been laid out in this article we posted for our initial Pomelo grant: https://medium.com/eostarter/eostarter-tokensale-platform-scope-of-work-4cb153c33ab6
+
+- Mockups/designs of any UI components: 
+  The final UI components for the TokenSale platform are yet to be designed.
+  
+- Data models / API specifications of the core functionality
+  The Data Model can be observed in our current TokenSale Smart Contract MVP: https://github.com/eostarter/eostarter.sale-smart-contract
+
+- An overview of the technology stack to be used
+  This project features all the latest technologies and practices in the industry.
+- Hapi
+- Hasura
+- React
+- Kubernetes
+- EOSIO
+
+- Documentation of core components, protocols, architecture, etc. to be deployed
+  The core documentation for the platform requirements can be read here: https://github.com/eostarter/token-sale-platform
+  
+- PoC/MVP or other relevant prior work or research on the topic
+  We have a working DEMO of the SmartContract ready to test on Jungle at: https://tokensale.eosio.cr/
+  * Please contact us so that we can verify your Jungle account as either an Investor or a Project Owner (the 2 main users for the platform)
```

Comment:
Would it be possible to provide a "demo" Jungle account names, that way users can use "Watch Mode" to preview the demo site.

When logging in with normal Jungle account, the demo doesn't show any specific details to PoC/MVP (assuming this is by design?).